README.md

```diff
@@ -1,19 +1,24 @@
+<!-- START_METADATA
 ---
-title: Introduction
```

Comment:
uuh, don't we need this one? Does it magically default to the #Introduction on line 10?

Reply:
I'd like us to use the metadata in the PR I made.
```
---
title: Introduction to the Webhooks API
sidebar_label: Introduction
sidebar_position: 1
hide_table_of_contents: true
pagination_prev: null
pagination_next: null
---
```

Reply:
It does magically default to #Introduction. By default the title tag is actually ignored if there is a top level title in markdown (#). 

@rebekaburnett, I'll update!